src/actions/sftp/sftp.ts

```diff
@@ -32,7 +32,7 @@ export class SFTPAction extends Hub.Action {
         throw "Needs a valid SFTP address."
       }
       const data = request.attachment.dataBuffer
-      const fileName = request.formParams.filename || request.suggestedFilename() as string
+      const fileName = request.formParams.filename || request.suggestedFilename()
```

Comment:
since changes to `request.suggestedFilename` mean it no longer has a chance at being `undefined`, the linter said type assertion here is no longer necessary.